package.json

```diff
@@ -5,7 +5,7 @@
     "data_model_version": 1
   },
   "module": "dest/d4l.js",
-  "license": "BSD-3-Clause",
+  "license": "See license in LICENSE",
```

Comment:
Is that becoming a link? otherwise I'd state "See license in file _LICENSE_"

Reply:
The package.json does not support markdown, so this will not be a link and just follow the [example on the official npm site](https://docs.npmjs.com/files/package.json#license) and mention the filename.